src/main/solr/dssolr/conf/schema.xml

```diff
@@ -661,10 +661,33 @@ In solrconfig.xml the classic schema is defined with this:  <schemaFactory class
     <?description ?>
     <?example     ?>
   </field>
-  <!--TODO: Figure out what this field can contain-->
-  <field name="internal_program_structure_overlaps" type="string">
-    <?description ?>
-    <?example     ?>
+  <field name="internal_program_structure_overlaps" type="boolean">
+    <?description A boolean value that indicates if there are overlaps between this resource and other resources.?>
+    <?example     true?>
+  </field>
+  <field name="internal_program_structure_overlap_type_one_length_ms" type="plong">
+    <?description Indicates the length of an overlap of type one in ms.?>
```

Comment:
What is type one? As we don't know, please make it easier for future developers to investigate by stating what we /do/ know, i.e. that it is from the Preservica system.

Reply:
Very nice now!